beacon-chain/rpc/validator/status.go

```diff
@@ -188,8 +189,13 @@ func (vs *Server) depositBlockSlot(ctx context.Context, eth1BlockNumBigInt *big.
 	timeToInclusion := eth1UnixTime.Add(votingPeriodSeconds)
 
 	eth2Genesis := time.Unix(int64(beaconState.GenesisTime), 0)
-	eth2TimeDifference := timeToInclusion.Sub(eth2Genesis).Seconds()
-	depositBlockSlot := uint64(eth2TimeDifference) / params.BeaconConfig().SecondsPerSlot
+
+	if eth2Genesis.After(timeToInclusion) {
```

Comment:
This is a good candidate for a regression test

Reply:
Yes, correct @terencechain. I have added the test cases for depostBlockSlot function. Please take a look.